frontend/src/scenes/feature-flags/FeatureFlagCodeOptions.tsx

```diff
@@ -0,0 +1,251 @@
+import {
+    JSPayloadSnippet,
+    NodeJSPayloadSnippet,
+    PythonPayloadSnippet,
+    RubyPayloadSnippet,
+} from './FeatureFlagPayloadSnippets'
+import {
+    UTM_TAGS,
+    JSSnippet,
+    AndroidSnippet,
+    iOSSnippet,
+    ReactNativeSnippet,
+    NodeJSSnippet,
+    PythonSnippet,
+    RubySnippet,
+    APISnippet,
+    PHPSnippet,
+    GolangSnippet,
+    NodeLocalEvaluationSnippet,
+    PHPLocalEvaluationSnippet,
+    RubyLocalEvaluationSnippet,
+    PythonLocalEvaluationSnippet,
+    JSBootstrappingSnippet,
+    JSMultivariateSnippet,
+    AndroidMultivariateSnippet,
+    iOSMultivariateSnippet,
+    ReactNativeMultivariateSnippet,
+    PythonMultivariateSnippet,
+    RubyMultivariateSnippet,
+    PHPMultivariateSnippet,
+    GolangMultivariateSnippet,
+    NodeJSMultivariateSnippet,
+} from './FeatureFlagSnippets'
+
+const DOC_BASE_URL = 'https://posthog.com/docs/'
+const FF_ANCHOR = '#feature-flags'
+export const LOCAL_EVAL_ANCHOR = '#local-evaluation'
+export const BOOTSTRAPPING_ANCHOR = '#bootstrapping-flags'
+
+export interface InstructionOption {
+    value: string
+    documentationLink: string
+    Snippet: ({ flagKey }: { flagKey: string }) => JSX.Element
+    type: LibraryType
+}
+
+export enum LibraryType {
+    Client = 'Client',
+    Server = 'Server',
+}
+
+export const OPTIONS: InstructionOption[] = [
+    {
+        value: 'JavaScript',
+        documentationLink: `${DOC_BASE_URL}integrations/js-integration${UTM_TAGS}${FF_ANCHOR}`,
+        Snippet: JSSnippet,
+        type: LibraryType.Client,
+    },
+    {
+        value: 'Android',
+        documentationLink: `${DOC_BASE_URL}integrate/client/android${UTM_TAGS}${FF_ANCHOR}`,
+        Snippet: AndroidSnippet,
+        type: LibraryType.Client,
+    },
+    {
+        value: 'iOS',
+        documentationLink: `${DOC_BASE_URL}integrate/client/ios${UTM_TAGS}${FF_ANCHOR}`,
+        Snippet: iOSSnippet,
+        type: LibraryType.Client,
+    },
+    {
+        value: 'ReactNative',
+        documentationLink: `${DOC_BASE_URL}integrate/client/react-native${UTM_TAGS}${FF_ANCHOR}`,
+        Snippet: ReactNativeSnippet,
+        type: LibraryType.Client,
+    },
+    {
+        value: 'Node.js',
+        documentationLink: `${DOC_BASE_URL}integrations/node-integration${UTM_TAGS}${FF_ANCHOR}`,
+        Snippet: NodeJSSnippet,
+        type: LibraryType.Server,
+    },
+    {
+        value: 'Python',
+        documentationLink: `${DOC_BASE_URL}integrations/python-integration${UTM_TAGS}${FF_ANCHOR}`,
+        Snippet: PythonSnippet,
+        type: LibraryType.Server,
+    },
+    {
+        value: 'Ruby',
+        documentationLink: `${DOC_BASE_URL}integrations/ruby-integration${UTM_TAGS}${FF_ANCHOR}`,
+        Snippet: RubySnippet,
+        type: LibraryType.Server,
+    },
+    {
+        value: 'API',
+        documentationLink: `${DOC_BASE_URL}api/post-only-endpoints#example-request--response-decide-v2`,
```

Comment:
go to v3 now?

Reply:
no because the API docs for decide v3 yet isn't even in